extensions/amp-font/0.1/amp-font.js

```diff
@@ -78,13 +78,6 @@ const CACHED_FONT_LOAD_TIME_ = 100;
 
 export class AmpFont extends AMP.BaseElement {
 
-
-  /** @override */
-  prerenderAllowed() {
```

Comment:
Yeah I think so. I haven't really touched this other than dropping this function which is now default to `return true` so nothing really changed for this. This element doesn't even override `layoutCallback` so not sure why `prerenderAllowed` was there in the first place.